app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -36,6 +39,7 @@ import java.net.URISyntaxException
 class PocketVideoFragment : Fragment() {
 
     private lateinit var deferredVideos: PocketVideosDeferred
+    private var job: Job? = null
```

Comment:
This is something that occurred to me, but as we currently only have one long running task in this file, I didn't think it was worth the extra complexity.  That said, I don't feel strongly about it.  Would you like me to add it in?

Reply:
I feel the parent job approach is less complex because it separates concerns (i.e. doesn't mix job cancellation and job creation) and doesn't force anyone to refactor existing jobs to create new jobs.